src/common-signals.js

```diff
@@ -50,4 +50,9 @@ export const CommonSignals = {
    * The element has been unlaid out.
    */
   UNLOAD: 'unload',
+
+  /**
+   * The element's overflow element was clicked.
+   */
+  SIZE_CHANGED: 'size-changed',
```

Comment:
name is too generic single the signal only covers when `changeSize` method is called and not when other cases like CSS class is changed. Maybe `CHANGE_SIZE_END`? 

Also update the comment to reflect
```
Element's changeSize was called.
```
as it may happen without clicking overflow ( e.g. `attemptChangeSize` succeeding)

Reply:
Yup that sounds good. I will make that change in https://github.com/ampproject/amphtml/pull/18580 and rebase this PR.

Reply:
SG